src/seedu/addressbook/commands/CountCommand.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.commands;
+
+
```

Comment:
Stray line break? It is a good practice to check the diff of the PR and fix any problems before submitting it.